packs/google/requirements.txt

```diff
@@ -0,0 +1 @@
+-e git+https://github.com/Kami/Google-Search-API.git@stackstorm_patched#egg=Google-Search-API
```

Comment:
True, not a bad idea. Only if those sensors were easier to configure dynamically (aka triggers with parameters) :P